### ISSUE TITLE
Feature Request - Ability to show both original and replaced title

### ISSUE THREAD

**Issue by turpinator** [ajayyy/DeArrow#264]
I like the replaced titles, but also would like to read the original titles from the channels/creators.  I think of it kind of like title and its subheading.

Include the ability to show replaced or original first.
Ability to reduce the font size/style/etc of second text?

**Comment by robert-corlett** [comment]
Yes, that would be nice. Show the original title first, then show the alternative title on the next line in a slightly different font weight or color or whatever.
